src/Service/PagePurger.php

```diff
@@ -43,6 +44,24 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.x
+	 *
+	 * @param Pages $pages
+	 *
+	 * @return bool
+	 *
```

Comment:
There shouldn't be an empty line after the `@return` tag. Also, it'd be good to explain the return value (however, see my comment below).